diff -ruN speedy/openwebmail-abook.pl suidperl/openwebmail-abook.pl
--- speedy/openwebmail-abook.pl	Tue Mar 25 00:07:44 2003
+++ suidperl/openwebmail-abook.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_abook
+#!/usr/bin/suidperl -T
 #
 # openwebmail-abook.pl - address book program
 #
diff -ruN speedy/openwebmail-advsearch.pl suidperl/openwebmail-advsearch.pl
--- speedy/openwebmail-advsearch.pl	Tue Mar 25 00:08:12 2003
+++ suidperl/openwebmail-advsearch.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_advsearch
+#!/usr/bin/suidperl -T
 #
 # openwebmail-adsearch.pl - advanced search program
 #
diff -ruN speedy/openwebmail-cal.pl suidperl/openwebmail-cal.pl
--- speedy/openwebmail-cal.pl	Tue Mar 25 00:07:42 2003
+++ suidperl/openwebmail-cal.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_cal
+#!/usr/bin/suidperl -T
 #
 # openwebmail-cal.pl - calendar program
 #
diff -ruN speedy/openwebmail-folder.pl suidperl/openwebmail-folder.pl
--- speedy/openwebmail-folder.pl	Tue Mar 25 00:08:09 2003
+++ suidperl/openwebmail-folder.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_folder
+#!/usr/bin/suidperl -T
 #
 # openwebmail-folder.pl - mail folder management program
 #
diff -ruN speedy/openwebmail-main.pl suidperl/openwebmail-main.pl
--- speedy/openwebmail-main.pl	Tue Mar 25 00:07:41 2003
+++ suidperl/openwebmail-main.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_main
+#!/usr/bin/suidperl -T
 #
 # openwebmail-main.pl - message list browing program
 #
diff -ruN speedy/openwebmail-prefs.pl suidperl/openwebmail-prefs.pl
--- speedy/openwebmail-prefs.pl	Tue Mar 25 00:08:06 2003
+++ suidperl/openwebmail-prefs.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_prefs
+#!/usr/bin/suidperl -T
 #
 # openwebmail-prefs.pl - preference configuration, book editing program
 #
diff -ruN speedy/openwebmail-read.pl suidperl/openwebmail-read.pl
--- speedy/openwebmail-read.pl	Tue Mar 25 00:07:39 2003
+++ suidperl/openwebmail-read.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_read
+#!/usr/bin/suidperl -T
 #
 # openwebmail-read.pl - message reading program
 #
diff -ruN speedy/openwebmail-send.pl suidperl/openwebmail-send.pl
--- speedy/openwebmail-send.pl	Tue Mar 25 00:08:04 2003
+++ suidperl/openwebmail-send.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_send
+#!/usr/bin/suidperl -T
 #
 # openwebmail-send.pl - mail composing and sending program
 #
diff -ruN speedy/openwebmail-spell.pl suidperl/openwebmail-spell.pl
--- speedy/openwebmail-spell.pl	Tue Mar 25 00:07:38 2003
+++ suidperl/openwebmail-spell.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_spell
+#!/usr/bin/suidperl -T
 #
 # openwebmail-spell.pl - spell check program
 #
diff -ruN speedy/openwebmail-tool.pl suidperl/openwebmail-tool.pl
--- speedy/openwebmail-tool.pl	Tue Mar 25 00:08:02 2003
+++ suidperl/openwebmail-tool.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_tool
+#!/usr/bin/suidperl -T
 #
 # openwebmail-tool.pl - command tool for mail/event/notify/index...
 #
diff -ruN speedy/openwebmail-vdomain.pl suidperl/openwebmail-vdomain.pl
--- speedy/openwebmail-vdomain.pl	Tue Mar 25 00:08:16 2003
+++ suidperl/openwebmail-vdomain.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_vdomain
+#!/usr/bin/suidperl -T
 #
 # openwebmail-vdomain.pl - virtual domain user management
 #
diff -ruN speedy/openwebmail-viewatt.pl suidperl/openwebmail-viewatt.pl
--- speedy/openwebmail-viewatt.pl	Tue Mar 25 00:08:00 2003
+++ suidperl/openwebmail-viewatt.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_viewatt
+#!/usr/bin/suidperl -T
 #
 # openwebmail-viewatt.pl - attachment reading program
 #
diff -ruN speedy/openwebmail-webdisk.pl suidperl/openwebmail-webdisk.pl
--- speedy/openwebmail-webdisk.pl	Tue Mar 25 00:08:14 2003
+++ suidperl/openwebmail-webdisk.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_webdisk
+#!/usr/bin/suidperl -T
 #
 # openwebmail-webdisk.pl - web disk program
 #
diff -ruN speedy/openwebmail.pl suidperl/openwebmail.pl
--- speedy/openwebmail.pl	Tue Mar 25 00:07:57 2003
+++ suidperl/openwebmail.pl	Tue Mar 25 00:04:20 2003
@@ -1,4 +1,4 @@
-#!/usr/local/bin/speedy_suid -T -- -T/var/run/speedy_login
+#!/usr/bin/suidperl -T
 #################################################################
 #                                                               #
 # Open WebMail - Provides a web interface to user mailboxes     #
